internal/packages/internal/packagekickstart/presets/param.go

```diff
@@ -0,0 +1,96 @@
+package presets
+
+import (
+	"slices"
+
+	apiextensionsv1 "k8s.io/apiextensions-apiserver/pkg/apis/apiextensions/v1"
+	"k8s.io/apimachinery/pkg/apis/meta/v1/unstructured"
+)
+
+type ParametrizeOptions struct {
+	// Parametrize .metadata.namespace on all objects.
+	// .config.namespace will set the namespace on all objects and
+	// .config.namespaces.<name> can be used to override the name of a specifig namespace.
+	Namespaces bool
+
+	// Replicas adds a variable to set Deployment replicas.
+	// .config.deployments.<namespace>.<name>.replicas is setting replicas.
+	Replicas      bool
+	Tolerations   bool
+	NodeSelectors bool
+	Resources     bool
+	Env           bool
+	Images        bool
+}
+
+const (
+	namespaceParam     = "namespaces"
+	replicasParam      = "replicas"
+	tolerationsParam   = "tolerations"
+	nodeSelectorsParam = "nodeselectors"
+	resourcesParam     = "resources"
+	envParam           = "env"
+	imagesParam        = "images"
+	allParam           = "all"
+)
+
+func ParametrizeOptionsFromFlags(opts []string) ParametrizeOptions {
+	if slices.Contains(opts, "all") {
```

Comment:
Should we throw an error if `all` is passed together with any other value? It may hint an error in the command call. Wdyt?